pkg/ebpf/events_derived.go

```diff
@@ -145,6 +151,27 @@ func deriveDetectHookedSyscall(t *Tracee) deriveFn {
 	}
 }
 
+// deriveNetPacket driving net_packet from net events with 'metadata' arg
+func deriveNetPacket() deriveFn {
```

Comment:
Should be a `func deriveNetPacket(t *Tracee) deriveFn {` for conformity.

Reply:
yeah i wasn't sure, because `t` would not be used in this func.

Reply:
what did we decide? should i do it like that although `t` is not used in the function?

Reply:
Yes please. Not because we use it or not, just because when we outline (with IDE plugins or in general terms) the function, it should be part of the `Tracee` "class" (if we can say this in Go =D)).

Reply:
I'm doing this and merging your commits. Will flag the PR being merged here (with your commits) and close this one.

Reply:
Nah, linters complain, its okay then. Thanks, sorry for the noise in this one.